src/cdomains/apron/linearTwoVarEqualityDomain.apron.ml

```diff
@@ -153,31 +195,70 @@ module EqualitiesConjunction = struct
 
   exception Contradiction
 
-  let meet_with_one_conj ts i (var, b) =
+  let meet_with_one_conj ts i (var, offs, divi) =
+    let (var,offs,divi) = Rhs.canonicalize (var,offs,divi) in (* make sure that the one new conj is properly canonicalized *)
     let res =
-      let subst_var tsi x (vart, bt) =
+      let subst_var tsi x (vary, o, d) =
```

Comment:
It seems `tsi` is only ever decomposed and not otherwise referenced, so directly turning this into a tuple here may make it easier to read...